src/lib/graphql_basic_scalars/testing.ml

```diff
@@ -73,7 +73,16 @@ struct
     test_query schema () "{ test }" (fun response ->
         [%test_eq: G.t] value (S.parse @@ get_test_field response) )
 
-  let%test_unit "test" =
+  let test_query () =
     Core_kernel.Quickcheck.test G.gen ~sexp_of:G.sexp_of_t
       ~f:query_server_and_compare
 end
+
+module Make_test
+    (S : Json_intf_any_typ with type ('a, 'b) typ := ('a, 'b) Graphql.Schema.typ)
+    (G : Test_Intf with type t = S.t) =
+struct
+  include Produce_test (S) (G)
+
+  let%test_unit "test" = test_query ()
```

Comment:
Don't we want to move it outside?

Reply:
Yes. For `Pickles` only, though. This way I'm preventing any change to other libraries that might use `Make_test`.